gpjax/gps.py

```diff
@@ -20,6 +18,46 @@
 from .types import Array, Dataset
 from .utils import I, concat_dictionaries
 
+from .config import get_defaults
+
+DEFAULT_JITTER = get_defaults()["jitter"]
+
+########################################################
+### Gaussian process (GP) notation used in the code: ###
+########################################################
+
+#   - x     for the train inputs
+#   - y     for train labels
+#   - t     for test inputs
```

Comment:
I don't like using `t` for test points; it feels too ambigous with time indices. Can we use `xtrain` and `xtest`, or `x` and `xstar`?